performance/src/gatling/simulations/org/pmiops/databrowser/Configuration.scala

```diff
@@ -0,0 +1,17 @@
+package org.pmiops.databrowser
+
+object Configuration {
+
+  val env: String = System.getProperty("env", "stable")
+  val defaultPause: Int = 3
+  val userAgentHeader: String = "Gatling Performance Testing"
+  val defaultUrl: String = {
+    env match {
+      case "prod" => "https://databrowser.researchallofus.org"
+      case "stable" => "https://www.databrowser.stable.fake-research-aou.org"
+      case "staging" => "https://www.databrowser.staging.fake-research-aou.org"
```

Comment:
Per my comment further down, we should switch this to use the API endpoints instead, here are the correct values:

https://public.api.researchallofus.org
https://public.api.stable.fake-research-aou.org
https://api-dot-aou-db-staging.appspot.com
https://api-dot-aou-db-test.appspot.com

Reply:
Will do. I'll also do some scenarios refactoring so the focus is on APIs.

Reply:
Done.